pkg/cli/kubernetes/kubernetes.go

```diff
@@ -117,6 +121,30 @@ func EnsureNamespace(ctx context.Context, client k8s.Interface, namespace string
 	return nil
 }
 
+// DeleteNamespace delete the specified namespace.
+func DeleteNamespace(ctx context.Context, client k8s.Interface, namespace string) error {
+	gracePeriodSeconds := int64(10)
```

Comment:
Is there a specific motivation for 10 seconds? If we don't have a reason to configure a different value we should stick with the default. 

Context: https://kubernetes.io/docs/reference/kubectl/generated/kubectl_delete/